app/models/derived_variables/lettings_log_variables.rb

```diff
@@ -306,9 +306,9 @@ def property_vacant_days
     return unless startdate
 
     if mrcdate.present?
-      (startdate - mrcdate).to_i / 1.day
+      startdate.to_date - mrcdate.to_date
     elsif voiddate.present?
-      (startdate - voiddate).to_i / 1.day
```

Comment:
FYI the bug comes from doing this calculation with a datetime. If the startdate is in summer time and the mrcdate / voiddate is not then the subtraction gives a number slightly under the expected number of days (i.e. an hour short), and the integer division gives the floor.